src/Roslyn.Diagnostics.Analyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -2,6 +2,9 @@
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
 RS0040 | RoslynDiagnosticsReliability | Warning | DefaultableTypeShouldHaveDefaultableFieldsAnalyzer
+RS0043 | RoslynDiagnosticsMaintainability | Warning | DoNotCallGetTestAccessor
+RS0044 | RoslynDiagnosticsMaintainability | Hidden | CreateTestAccessor
+RS0045 | RoslynDiagnosticsMaintainability | Hidden | ExposeMemberForTesting
```

Comment:
I think both these hidden diagnostics should be removed and we should convert these into refactorings. @jcouv already added a refactoring to this package, so hopefully this should be simple even from testing perspective.

Reply:
Yes, they could be converted. Right now I just need these to work though and the two approaches are indistinguishable for users.